src/main/resources/config/application.yaml

```diff
@@ -51,3 +51,7 @@ spring:
       hibernate.temp.use_jdbc_metadata_defaults: false
       hibernate.jdbc.time_zone: UTC
       hibernate.default_schema: stayeasy
+api:
+  security:
+    token:
+      secret: ${JWT_SECRET:my-secret-key}
```

Comment:
Se for uma variavel de ambiente que precisa executar algum script na maquina, substituia essa abordagem por deixar hardcoded aqui para que todos possam executar o codigo sem precisar executar esse script